templates/default/nginx.conf.erb

```diff
@@ -4,7 +4,7 @@ upstream disclosurebackend {
 
 server {
   listen 80;
-  server_name opencal.opendisclosure.io;
+  server_name opencal.opendisclosure.io disclosure.caciviclab.org;
```

Comment:
This new hostname will need to be added to settings_local.py's `ALLOWED_HOSTS` as well, but that isn't in version control.